ospf.h

```diff
@@ -110,9 +111,12 @@
 #define LS_OPAQUE_GRACE_TLV_REASON_SW_UPGRADE  2 /* rfc3623 */
 #define LS_OPAQUE_GRACE_TLV_REASON_CP_SWITCH   3 /* rfc3623 */
 
-#define LS_OPAQUE_RI_TLV_CAP             1 /* draft-ietf-ospf-cap-03 */
-#define LS_OPAQUE_RI_TLV_HOSTNAME        7 /* rfc5642 */
-#define LS_OPAQUE_RI_TLV_SID_LABEL_RANGE 9 /* rfc8665 */
+#define LS_OPAQUE_RI_TLV_CAP               1 /* draft-ietf-ospf-cap-03 */
```

Comment:
rfc7770 ?

Reply:
Yes -  draft-ietf-ospf-cap-03 became rfc4970 which later got obsoleted by rfc7770
Here is the history -> https://datatracker.ietf.org/doc/search?name=+draft-ietf-ospf-cap-03&sort=&rfcs=on&activedrafts=on&by=group&group=

Reply:
Yup!